assets/airports/kric.json

```diff
@@ -0,0 +1,9359 @@
+            ]
+        },
+        "KALLI7": {
+            "icao": "KALLI7",
+            "name": "Kalli Seven",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SUGGR", "A50+"]],
+                "KRIC16": [["^_RW161DME", "A007+"], ["ITTEM", "A50+"]],
+                "KRIC20": [["^_RW201DME", "A007+"], ["ITTEM", "A50+"]],
+                "KRIC34": [["^_RW341DME", "A007+"], ["SUGGR", "A50+"]]
+            },
+            "body": ["KALLI"],
+            "exitPoints": {
+                "DRAIK": ["READE", "DRAIK"],
+                "READE": ["READE"],
+                "SANNY": ["READE", "DRAIK", "SANNY"]
+            },
+            "draw": [
+                ["_RW021DME", "SUGGR", "_RW341DME"],
+                ["SUGGR", "KALLI", "READE*", "DRAIK*", "SANNY*"],
+                ["_RW161DME", "ITTEM", "_RW201DME"],
+                ["ITTEM", "KALLI"]
+            ]
+        },
+        "LUCYL6": {
+            "icao": "LUCYL6",
+            "name": "Lucyl Six",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], ["SJAYY", "A50+"], "KAMMI"],
+                "KRIC16": [["^_RW161DME", "A007+"], ["KEGME", "A50+"], "HAYYU"],
+                "KRIC20": [["^_RW201DME", "A007+"], ["KEGME", "A50+"], "HAYYU"],
+                "KRIC34": [["^_RW341DME", "A007+"], ["SJAYY", "A50+"], "KAMMI"]
+            },
+            "body": ["LUCYL"],
+            "exitPoints": {
+                "LUCYL": []
+            },
+            "draw": [
+                ["_RW021DME", "SJAYY", "_RW341DME"],
+                ["SJAYY", "KAMMI", "LUCYL*", "HAYYU", "BUNKK", "KEGME"],
+                ["_RW161DME", "KEGME", "_RW201DME"]
+            ]
+        },
+        "READE7": {
+            "icao": "READE7",
+            "name": "Reade Seven",
+            "rwy": {
+                "KRIC2": [["^_RW021DME", "A007+"], "_RIC26410"],
+                "KRIC16": [["^_RW161DME", "A007+"], "_RIC26410"],
+                "KRIC20": [["^_RW201DME", "A007+"], "_RIC26410"],
+                "KRIC34": [["^_RW341DME", "A007+"], "_RIC26410"]
+            },
+            "body": ["CARML", "READE"],
+            "exitPoints": {
+                "DRAIK": ["DRAIK"],
+                "LYH": ["LYH"],
+                "MOL": ["MOL"],
+                "SANNY": ["DRAIK", "SANNY"]
+            },
+            "draw": [
+                ["_RW021DME", "_RIC26410", "_RW341DME"],
+                ["_RIC05010", "COLIN"],
+                ["_RW161DME", "_RIC26410", "_RW201DME"],
+                ["_RIC26410", "CARML", "READE", "DRAIK*", "SANNY*"],
+                ["LYH*", "READE*", "MOL*"]
+            ]
+        }
+    },
+    "stars": {
+       "DUCXS5": {
+            "icao": "DUCXS5",
+            "name": "Ducks Five",
+            "entryPoints": {
+                "KELCE": [["KELCE", "A110|S250"]],
+                "NEAVL": [["NEAVL", "A110|S250"], "PULDY"]
+            },
+            "body": ["THINA"],
+            "rwy": {
```

Comment:
Although the chart states that vectors should be expected prior to THINA when landing North, aircraft would be expected to continue to DUCXS if the vectors aren't given in time.